tests/test_blotter.py

```diff
@@ -112,6 +114,20 @@ def test_blotter_order_types(self, style_obj, expected_lmt, expected_stp):
         self.assertEqual(result.limit, expected_lmt)
         self.assertEqual(result.stop, expected_stp)
 
+    def test_blotter_eod_cancellation(self):
+        blotter = Blotter('minute', self.env.asset_finder)
+
+        blotter.order(24, 100, MarketOrder(), cancel_policy=EODCancel())
+        blotter.order(25, 100, MarketOrder(), cancel_policy=EODCancel())
+
+        self.assertEqual(len(blotter.new_orders), 2)
+        for order in blotter.new_orders:
+            self.assertEqual(order.status, ORDER_STATUS.OPEN)
+
+        blotter.execute_cancel_policy(None, DAY_END)
```

Comment:
This should also asssert that if we execute the cancel policy **not** on DAY_END, that we don't cancel the order.